### ISSUE TITLE
404 error after first new milestone

### ISSUE THREAD

**Issue by projectplant** [Coren/redmine_advanced_roadmap_v2#37]
Hi,

With a fresh, new Redmine 3.0 and the only advanced roadmap v2 plugin enabled, I got an 404 error when I add a new "milestone 1" (assoziate with the only "version 1").
![auswahl_004](https://cloud.githubusercontent.com/assets/11331755/6505443/9b28f7a8-c342-11e4-9f1c-b2daaa0c58e4.png)

![auswahl_005](https://cloud.githubusercontent.com/assets/11331755/6505468/e6c96e4a-c342-11e4-8a6c-ba18e23633b6.png)

production.log:
```
Started GET "/projects/test-mbu/milestones/new?back_url=" for 192.168.1.2 at 2015-03-05 14:20:54 +0100
Processing by MilestonesController#new as HTML
  Parameters: {"back_url"=>"", "project_id"=>"test-mbu"}
  Current user: admin (id=1)
  Rendered plugins/advanced_roadmap_v2/app/views/milestones/_form.html.erb (27.6ms)
  Rendered plugins/advanced_roadmap_v2/app/views/milestones/new.html.erb within layouts/base (32.2ms)
  Rendered plugins/advanced_roadmap_v2/app/views/hooks/_stylesheet.html.erb (0.7ms)
Completed 200 OK in 142ms (Views: 112.4ms | ActiveRecord: 6.9ms)
Started POST "/projects/test-mbu/milestones" for 192.168.1.2 at 2015-03-05 14:21:02 +0100
Processing by MilestonesController#create as HTML
  Parameters: {"utf8"=>"✓", "authenticity_token"=>"3x52CoI8ZmJS/uDt3oF5bNe7ZLjv+a7SDEYQNg7N1UEyGPCPW+LVSWff6NZDjgO6EPVFPXRdbB4M5Ivq4J2n0Q==", "milestone"=>{"name"=>"Meilenstein 1", "description"=>"", "effective_date"=>""}, "versions"=>["1"], "commit"=>"Anlegen", "project_id"=>"test-mbu"}
  Current user: admin (id=1)
WARNING: Can't mass-assign protected attributes for Milestone: name, description, effective_date

Missing template, responding with 404
  Rendered common/error.html.erb within layouts/base (0.6ms)
  Rendered plugins/advanced_roadmap_v2/app/views/hooks/_stylesheet.html.erb (0.7ms)
Completed 404 Not Found in 90ms (Views: 41.7ms | ActiveRecord: 5.5ms)
```

Environment:
  Redmine version                3.0.0.stable.14045
  Ruby version                   2.2.0-p0 (2014-12-25) [x86_64-linux]
  Rails version                  4.2.0
  Environment                    production
  Database adapter               Mysql2
SCM:
  Subversion                     1.8.8
  Git                            1.9.1
  Filesystem                     
Redmine plugins:
  advanced_roadmap_v2            2.2.1

**Comment by Coren** [comment]
Hi,

Would you please try advanced roadmap v2 v2.3.0 ?

Regards,

**Comment by projectplant** [comment]
It's fixed with 2.3.1 ... but as result roadmap is broken, see #39.
